help.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<style type="text/css">
+body {
+  min-width: 100px; 
+  position: relative;
+  vertical-align:middle;
+}
+
+.label-container {
+  padding-top: 8px;
+  padding-bottom: 10px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.label-title {
+  font-weight: bold;
+  padding-right: 5px;
+}
+
+.origin-image {
+  margin-top: -1px;
+}
+
+.cache-image {
+  margin-top: -1px;
+}
+
+.debug-container {
+  display: flex;
+  flex-direction: row;
+  padding-top: 15px;
+  padding-bottom: 5px;
+}
+
+</style>
+
+<meta charset="UTF-8">
+<html><head><title>Signed Exchange (SXG) Validator</title></head><body>
+<h2>Welcome to the help page of the Signed Exchange Validator Chrome Extension.</h2>
+<p>
+    This tool is intended to help website administrators to determine the status
+    of their Signed Exchange deployments. It may include the status of the origin
+    URL, its corresponding Google SXG Cache URL as well as the Google SXG Certificate
+    URL.
+</p>
+<div class="label-container">
+  <div class="origin-image" alt="Origin URL State" id="originimg">
+   The different images the tool displays indicates the status of either the origin, cache or cert.
+   <br><br>
+   ... means it's processing.
+   <br>
+   ⌛ means not in cache yet.
+   <br>
+   ✅ means everything's ok.
+   <br>
+   ❌ means there's an error.
+   <br>
+  </div>
+</div>
+<table border="1" cellpadding="3" style="border-collapse:collapse;">
+  <tr>
+    <td style="min-width:100px" nowrap>URL</td>
+    <td align="left"><span id="url" alt="Origin URL">The URL of the SXG's origin, cache or SXG certificate.</span></td>
```

Comment:
Why `alt=` here? AFAIK that should be used for screenreaders (e.g. for image tags), but they can read the inner text just fine. (Ditto below.)

Reply:
Cut & paste. Removed.